modules/cluster/main.tf

```diff
@@ -52,14 +52,14 @@ module "vpc" {
 // See https://github.com/terraform-aws-modules/terraform-aws-eks
 // ----------------------------------------------------------------------------
 module "eks" {
-  source           = "terraform-aws-modules/eks/aws"
-  version          = "10.0.0"
-  cluster_name     = var.cluster_name
-  cluster_version  = var.cluster_version
-  subnets          = module.vpc.public_subnets
-  vpc_id           = module.vpc.vpc_id
-  enable_irsa      = true
-  worker_groups    = [
+  source          = "terraform-aws-modules/eks/aws"
+  version         = "12.1.0"
```

Comment:
This one is interesting. I guess here the logic from above does not apply. I would think upgrading here makes sense.

Reply:
Yes, I also think it makes sense :) The reasoning being that we dont use bare terraform resources to provision aws resources, instead we depend on these modules. So, bumping up the modules, is the only way for us to get some of the bug fixes. 

Most of the dependency issues are resolved in later versions of the vpx module for instance :)

Reply:
So you are saying that the VPC destruction works after the upgrade, even if you have a Jenkins X install? If so, a big +1.